src/TransitorySensitivities.jl

```diff
@@ -149,52 +149,58 @@ end
 function normalizefulltransitorysensitivities!(dSdt,sim::Simulation,t)
     y = sim.sol(t)
     @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
-    dSdt .*= sim.p'
+    #only fully normalize rate coefficient parameters
+    dSdt[:,sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]] .*= sim.domain.p[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]]'
     @views dSdt[sim.domain.indexes[1]:sim.domain.indexes[2],:] ./= ns
-    return dSdt
+    return dSdt[vcat(Array(sim.domain.indexes[1]:sim.domain.indexes[2]),[i for (j,i) in enumerate(sim.domain.indexes) if j > 2]),sim.domain.parameterindexes[1]:sim.domain.parameterindexes[2]]
 end
 
 function normalizefulltransitorysensitivities!(dSdt,ssys::SystemSimulation,t)
     y = ssys.sol(t)
-    dSdt .*= ssys.p'
     for sim in ssys.sims
+        #only fully normalize rate coefficient parameters
+        @views dSdt[:,sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]] .*= ssys.p[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]]'
         @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
-        @views dSdt[sim.domain.indexes[1]:sim.domain.indexes[2],:] ./= ns
+        @views dSdt[vcat(Array(sim.domain.indexes[1]:sim.domain.indexes[2]),[i for (j,i) in enumerate(sim.domain.indexes) if j > 2]),:] ./= ns
     end
     return dSdt
 end
 
 function normalizeparamtransitorysensitivities!(dSdt,sim::Simulation,t,ind)
     y = sim.sol(t)
     @views ns = y[sim.domain.indexes[1]:sim.domain.indexes[2]]
-    dSdt .*= sim.p[ind]
+    if ind <= length(sim.species)
+        dSdt .*= sim.domain.p[ind]
```

Comment:
Why are we doing this but not `dSdt[:,sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]] .*= sim.domain.p[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]]'`? I'm confused why sometimes we multiply p on the whole dSdt and sometimes we do it on the reaction parts

Reply:
Also why sometimes we use `dSdt[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]] .*= ssys.domain.p[sim.domain.parameterindexes[1]+length(sim.species):sim.domain.parameterindexes[2]]` and sometimes this?

Reply:
Okay, fixed that line. The difference is the dSdt calculation this is for only the calculations for individual parameters.